xtra-libp2p-offer/Cargo.toml

```diff
@@ -14,7 +14,7 @@ thiserror = "1"
 tokio = { version = "1", features = ["rt-multi-thread", "macros", "sync", "net"] }
 tokio-tasks = { path = "../tokio-tasks" }
 tracing = "0.1"
-xtra = { version = "0.6", features = ["metrics", "timeout", "tracing"] }
+xtra = { version = "0.6" }
```

Comment:
no need to add "instrumentation" to this one?

Reply:
`instrumentation` doesn't affect the public API, so only binaries really care about it being enabled I think.